docs_src/tutorials/kubernetes.md

```diff
@@ -0,0 +1,60 @@
+# Kubernetes
+
+This tutorial uses [Helm](https://helm.sh/docs/intro/quickstart/) to install
+Scaphandre, Prometheus and Grafana.
+
+## Install Scaphandre
+
+First we install Scaphandre which runs as a daemon set which creates a pod on
+each node for collecting the metrics.
+
+    helm install scaphandre helm/scaphandre
```

Comment:
Does that suppose we rely on a local chart ? Maybe we should add the step to download the right chart on the right location first ?

Reply:
Yes its using the local chart but that's not ideal. 

The central Helm charts repo is now deprecated and not accepting new charts. https://github.com/helm/charts#helm-charts

The new pattern is to have a charts repo per org e.g. https://github.com/grafana/helm-charts and then link it to https://artifacthub.io/ for discoverability.

It should be possible to use something like GitHub Actions to push new versions of the chart to the repo. Happy to help with that if I can.

Reply:
This is fine as a first version.

We may create an FR after that to setup a helm chart repository for scaphandre.